marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -69,6 +71,8 @@ mkWalletState pubKeyHash =
   , menuOpen: false
   , screen: ContractsScreen Running
   , card: Nothing
+  , templateState: Template.defaultState
+  , contractState: Contract.defaultState
```

Comment:
Could wrap both of these in a subtype in a `Maybe` or something like that, because we only have `templateState` and `contractState` either both at the same time or none

Reply:
As above.